include/message_filters/subscriber.h

```diff
@@ -140,9 +181,24 @@ class Subscriber : public SubscriberBase, public SimpleFilter<M>
    *
    * \param node The rclcpp::Node::SharedPtr to use to subscribe.
    * \param topic The topic to subscribe to.
-   * \param qos (optional) The rmw qos profile to use to subscribe
+   * \param qos (optional) The qos profile to use to subscribe
```

Comment:
Not sure if this is better or worse, this signature is still using the rmw version.

Reply:
I mantained